prow/cmd/deck/static/script.js

```diff
@@ -693,3 +692,18 @@ function createIcon(iconString, tooltip = "") {
 
     return container;
 }
+
+/**
+ * Returns the duration between two moments
+ * @param {date time} startTime
+ * @param {data time} completionTime 
+ */
+
+ function duration(startTime, completionTime) {
+     
+    var startMoment = new moment(startTime),
+        completionMoment = new moment(completionTime),
+        d = moment.duration(completionMoment.diff(startMoment));
+
+    return d.hours() + "h" + d.minutes() + "m" + d.seconds() + "s";
```

Comment:
`d.format()` ?

Reply:
Duration object seems not to have `format()`.

Reply:
Yeah, see https://github.com/moment/moment/issues/1048.

This is wrong for jobs that take too long, and it will look weird to have a bunch of 0h jobs.

    var out = d.seconds() + 's';    
    if (d.asHours() >= 1) 
        out = Math.floor(d.asHours()) + 'h' + d.minutes() + 'm' + out;
    else if (d.minutes() >= 1) 
        out = d.minutes() + 'm' + out;
    return out;